### PR TITLE
Add basics/loops.md

### PR DESCRIPTION
**Pull request by SMietzner**

### CODE REVIEW

File: basics/loops.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    while (condition) {
+        foo();
+    }
+
+### 2) `do ... while`
+
+Die `do .. while`-Schleife führt den gegebenen Code-Block aus,
+solange eine bestimmte Bedingung erfüllt ist. Aber im Gegensatz
+zur `while`-Schleife wird der Code-Block ausgeführt, bevor 
+die Bedingung das erste Mal abgeprüft wird.
```
Comment (wilzbach) [inline basics/loops.md]: s/abgeprüft/geprüft/

**Review by wilzbach — CHANGES_REQUESTED** [review]
LGTM minus einem ganz kleinem nit

**Review by wilzbach — APPROVED** [review]

**Comment by wilzbach** [comment]
Thanks a lot for your hard work!
I am on the side of rather merging early than never, thus I will now start to merge a couple of your PRs.
It's a lot easier to build upon existing work and just correct a small typo (and also easier for us to review).

As mentioned before, unfortunately auto-deploy doesn't work at the moment.

**Comment by SMietzner** [comment]
> It's a lot easier to build upon existing work and just correct a small typo (and also easier for us to review).

Agreed, I share your opinion on that.

**Pull request merged**